tests/test_she_evolve.py

```
@@ -0,0 +1,120 @@
+    ExperimentSpec,
+    Hypothesis,
+    plan_evolution,
+)
+from she.incident import Incident, IncidentState
+from she.sandbox import plan_repair_sandbox
+from she.verify import DUAL_GATES, plan_verification
+
+FIXED = datetime(2026, 8, 28, 5, 0, tzinfo=UTC)
+
+
+def _inc(**overrides) -> Incident:
```

Ruff (0.16.2)

[warning] Missing type annotation for `**overrides`

(ANN003)